### ISSUE TITLE
error when cast type 

### ISSUE THREAD

**Issue by godshow** [nicklockwood/GLView#25]
/GLView/Images/GLImage.m:289:60: Cast from 'const void *' to 'PVRTextureHeaderV2 *' drops const qualifier

**Comment by natasha1403** [comment]
I am also getting similar error? Could you find fix for the same?

**Comment by godshow** [comment]
@natasha1403 
the GLModelDemo example is running on my iphone, make a  deep copy of the const varible.
i use memcpy() function.

**Comment by natasha1403** [comment]
Can you please tell can I deep copy, I am new to xcode and objective C

**Comment by godshow** [comment]
@natasha1403 
like this 
```
void *dst = malloc([data length]);
memcpy(dst, [data bytes], [data length]);
PVRTextureHeaderV2 *header = (PVRTextureHeaderV2 *)dst;
```

**Comment by natasha1403** [comment]
but memcpy is used for copying only variable with type void. I am getting different errors as well:

/Users/natasha/Downloads/GLView-master/GLView/Models/GLModel.m:233:59: Sending 'const NSString *__strong' to parameter of type 'NSString * _Nonnull' discards qualifiers

**Comment by natasha1403** [comment]
I am getting error in every class one after the other.. is there an end to these issues??

**Comment by godshow** [comment]
@natasha1403 

* NO, memcpy() operate memory ,so anything is ok.

* you create a new varible to receive the const NSString, then send it

**Comment by natasha1403** [comment]
Thank you so much, finally this thing worked for me. But, none of the objs are loading, only demon got loaded for me. Did you face a similar issue?

**Comment by nicklockwood** [comment]
Fixed in 1.6.2
